editor/plugins/org.fusesource.ide.projecttemplates/plugin.xml

```diff
@@ -186,7 +186,7 @@
       
       	<project-facet-version
             facet="jst.camel"
-            version="2.15.1.redhat-621084">
+            version="1.0">
```

Comment:
I'd say we won't have an issue here because we will always use x.y notation while the old ones used x.y.z or even x.y.z.qualifier.